tests/fixtures/search_results/valid_search_by_subject.json

```diff
@@ -1 +1,83 @@
+                    "Abbot Mortimer"
+                ],
+                "title": "Experiments on Plant Hybridization",
+                "doi": "10.26311/fake-doi",
+                "description": "\nThis Garden houses sophisticated ML pipelines for Big Pea Data extraction\nand classification. It consists of a 2-hectare plot behind the monastery,\nand a 30,000-plant dataset.\n",
+                "resourceTypeGeneral": "Other",
+                "publisher": "Garden",
+                "year": "1863",
+                "language": "en",
+                "tags": [],
+                "version": "0.0.1",
+                "pipelines": [
+                    {
+                        "title": "Concept: a pipeline for soup",
+                        "authors": [
+                            "Mendel, Gregor"
+                        ],
+                        "steps": [
+                            {
+                                "func": "split_peas: (ps: List) -> List[tuple]",
+                                "authors": [
+                                    "Sister Constance"
+                                ],
+                                "contributors": [],
+                                "title": "split_peas",
+                                "description": null,
+                                "input_info": null,
+                                "output_info": null
+                            },
+                            {
+                                "func": "make_soup: (splits: List[tuple]) -> __main__.Soup",
+                                "authors": [
+                                    "Friar Hugo"
+                                ],
+                                "contributors": [],
+                                "title": "make_soup",
+                                "description": null,
+                                "input_info": null,
+                                "output_info": null
+                            },
+                            {
+                                "func": "rate_soup: (soup_sample: __main__.Soup) -> float",
+                                "authors": [
+                                    "Abbot Mortimer"
+                                ],
+                                "contributors": [
+                                    "Friar Hugo",
+                                    "Sister Constance"
+                                ],
+                                "title": "rate_soup",
+                                "description": null,
+                                "input_info": "a spoonful of Soup object",
+                                "output_info": null
+                            }
+                        ],
+                        "contributors": [
+                            "Sister Constance",
+                            "Abbot Mortimer",
+                            "Friar Hugo"
+                        ],
+                        "doi": null
```

Comment:
let's make sure the "valid search result" has a DOI for the pipeline, too, even though we don't currently instantiate a pipeline straight from the search record